packages/app/src/App.tsx

```diff
@@ -93,9 +98,60 @@ import { ApacheAirflowPage } from '@backstage/plugin-apache-airflow';
 import { PermissionedRoute } from '@backstage/plugin-permission-react';
 import { catalogEntityCreatePermission } from '@backstage/plugin-catalog-common';
 
+const highlightOverride = {
+  BackstageHighlightedSearchResultText: {
+    highlight: {
+      color: 'inherit',
+      backgroundColor: 'inherit',
+      fontWeight: 'bold',
+      textDecoration: 'underline',
+    },
```

Comment:
Is this something everyone that wants highlighting would need to add, or is it just an example of how to override it? I'm feeling it's a bit much code in the example `App.tsx` tbh. Perhaps it's something we could document in the theme docs instead?

If it's something everyone has to add then I think it's best to look for a solution that doesn't require changes to the app, but I'm assuming that's not the case given the create-app changes.

Reply:
Or perhaps if bloat in `App.tsx` is the concern, perhaps we could break out the theme stuff into a separate file and just import it in?  A section in the "how to guides" of the search docs could also work.  I'm open to any/all.  Whaddya think, @Rugvip?

Reply:
I added a section in the "how to guides" with some info/reference to theme docs on how to apply this customization

Reply:
@kuangp If it's in the docs, could we remove it from here now? I agree with the bloat, and it only means a risk of drifting away from the central theme and missing (possibly broken) changes there, if we can't see them in the example app.

Reply:
makes sense! updated!